### PR TITLE
TF modules for Zone_base firewall,feature policy and IPS policy . modifiy main.tf

### PR DESCRIPTION
**Pull request by narasing01**
@rrahimm  please review and share the review comments

### CODE REVIEW

File: sdwan_policy_objects.tf [hunk 1]
```diff
@@ -175,3 +175,24 @@ resource "sdwan_region_list_policy_object" "region_list_policy_object" {
...
+      vpn = e
+    }],
+    [for e in try(each.value.interfaces, []) : {
+      interface = e
+    }]
+  )
+}
+
+resource "sdwan_data_fqdn_prefix_list_policy_object" "fqdn_prefix_list_policy_object" {
+  for_each = { for p in try(local.policy_objects.fqdn_data_prefix_lists, {}) : p.name => p }
```
Comment (rrahimm) [inline sdwan_policy_objects.tf:193]: the key name is wrong >> fqdn_data_prefix_lists

it should be fqdn_lists

**Review by rrahimm — CHANGES_REQUESTED** [review]
the key name is wrong >> fqdn_data_prefix_lists

it should be fqdn_lists

**Comment by narasing01** [comment]
@rrahimm  review comments incorporate.

**Comment by narasing01** [comment]
@rrahimm  added the dependency in device templates for security policy

**Comment by rrahimm** [comment]
Hello @danischm 

Can you please review and approve

**Comment by rrahimm** [comment]
hello @tzarski0 

Can you review and approve this.

**Review by tzarski0 — APPROVED** [review]
approve

**Pull request merged**